src/app.js

```diff
@@ -0,0 +1,14 @@
+// src/app.js
+const express = require('express');
+const artistRouter = require('./routes/artist');
+
+const app = express();
+const cors = require('cors');
+
+app.use(cors());
+
+app.use(express.json());
+
+app.use('/artist', artistRouter);
```

Comment:
Blank lines could be removed here since its such a small project
```
app.use(cors());
app.use(express.json());
app.use('/artist', artistRouter);
```